### DISCUSSION TITLE
Set default value by select query

### DISCUSSION THREAD

**Discussion by abualmazen** [serenity-is/Serenity#6764] [General]
Hi ,how i can set default value for som fields by select query ?

**Comment by minhhungit** [comment]
there is [DefaultValue(your_value)] attribute that you can use on row.cs

**Comment by abualmazen** [comment]
Thank you ,but i don't want to set fixed value 
i want the value com from other table using select query .

**Comment by minhhungit** [comment]
there is an example here:

https://demo.serenity.is/BasicSamples/DefaultValuesInNewDialog

https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Modules/Dialogs/DefaultValuesInNewDialog/DefaultValuesInNewDialogPage.ts

**Reply by abualmazen** [reply]
It worked 
thank you
